201-vnet-transitive-bgp/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sharedKey": {
+            "value": "mysharedkey1234"
```

Comment:
really, gen-unique should be alphanumeric (gen-password has non-alpha chars)... can you try it and if it fails we'll have a log (for a bug) and then we can rollback the change...